recipe/meta.yaml

```diff
@@ -13,12 +13,12 @@ source:
 
 build:
   number: 0
-  script: pip install --no-deps .
```

Comment:
@minrk I am compiling this info into a CFEP now so you don't have to repeat it :smile: 

@mingwandroid and @msarahan we can move the discussion to the CFEP. I'll ping you all there as soon as I have the PR.

@nehaljwani can you revert just this change so we can merge this? (If you are busy I can do that for you.)